test/parallel/test-repl-tab-complete.js

```diff
@@ -267,3 +267,14 @@ putIn.run(['.clear']);
 testMe.complete('.b', common.mustCall((error, data) => {
   assert.deepStrictEqual(data, [['break'], 'b']);
 }));
+
+const testNonGlobal = repl.start({
+  input: putIn,
+  output: putIn,
+  useGlobal: false
+});
+
+testNonGlobal.complete('I', common.mustCall((error, data) => {
+  assert.deepStrictEqual(data, [['Infinity', '', 'Int16Array', 'Int32Array',
+                                 'Int8Array', 'Intl'], 'I']);
```

Comment:
`Intl` will not be there if node binary is not built with intl support. May want to make the inclusion of Intl here conditional